web/models/task_list.ex

```diff
@@ -29,6 +31,10 @@ defmodule CodeCorps.TaskList do
         position: 4
       }
     ]
+
+    for task_list <- task_list_data do
+      TaskList.changeset(%TaskList{}, task_list)
+    end
   end
```

Comment:
@begedin Thanks for the comment! I'm not super familiar with idiomatic Elixir, but seems like a thing is to use a lot of `|>`s, so I'll refactor accordingly and push a commit with that change.